src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstBase;
+    private int secondBase;
+    private int heigh;
+
+    public IsoscelesTrapezoid(int firstBase, int secondBase, int heigh, Color color) {
+        super(color);
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.heigh = heigh;
+    }
+
+    @Override
+    public double getArea() {
+        return ((firstBase + secondBase) * heigh) / 2.0;
+    }
+
+    @Override
+    public String getFigureInfo() {
+        return "Figure: IsoscelesTrapezoid, area: " + getArea() + " sq. units, firstBase: "

```

Comment:
The method 'getFigureInfo' should use consistent formatting for the output string. Consider adding commas or semicolons between the attributes for better readability.